### PR TITLE
feat: enable restarting logic for testnet and mainnet

### PR DESCRIPTION
**Pull request by reliveyy**
This PR enables simnet restarting logic on testnet and mainnet.

### CODE REVIEW

**Comment by reliveyy** [comment]
After restarting lnd in testnet, it will constantly get the error below when executing `lncli -n {network} -c {chain} getinfo`
```
[lncli] unable to read macaroon path (check the network setting!): open /root/.lnd/data/chain/litecoin/testnet/admin.macaroon: no such file or directory
```

**Comment by kilrau** [comment]
We are restarting lnd **because** of that error. Since it fixes it on Simnet, can you check line by line what you do different here than on the simnet restart?

**Comment by kilrau** [comment]
You'll let us know when ready to be tested by getting this PR out of draft state @reliveyy

**Comment by raladev** [comment]
> * [ ]  Cli stuck in restart stage for testnet in neutrino mode.

Reproduced again:
[testnet.log](https://github.com/ExchangeUnion/xud-docker/files/4682646/testnet.log)
[macr_lndbtc.log](https://github.com/ExchangeUnion/xud-docker/files/4682647/macr_lndbtc.log)
[macr_lndltc.log](https://github.com/ExchangeUnion/xud-docker/files/4682648/macr_lndltc.log)
[macr_xud.log](https://github.com/ExchangeUnion/xud-docker/files/4682650/macr_xud.log)

It seems restart does not solve macroon problem in this case at all.

There is 2 problems:
- restart performed only once during the checking (see testnet.log). If i understand flow correctly, it should be performed for each checking.
- maybe create was not successful, therefore restart gives nothing (still checking this suggesstion)

**Comment by raladev** [comment]
One more note:
After my  manual restart it starts working (getinfo returns data instead of macroon error).

Can we try to implement following flow with retries:
1. Create wallet, wait 3 sec after screation
2. check lnd status-> return macron error - > Trigger restart -> wait 10 sec
3. check lnd status -> if it returns macroon err -> try 2 more times  and Timeout if it does not help
4. if sucess -> restart xud and move on.

File: images/utils/launcher/__init__.py [hunk 1]
```diff
@@ -111,14 +108,18 @@ def auto_unlock(self):
     def close_other_utils(self):
         CloseOtherUtilsAction(self.config.network, self.shell).execute()
 
+    def warm_up(self):
+        WarmUpAction(self.node_manager).execute()
```
Comment (kilrau) [inline images/utils/launcher/__init__.py:112]: Here

**Pull request merged**